docs/source/usage/index.rst

```diff
@@ -22,11 +22,17 @@ AiiDAlab launch
 To run AiiDAlab on your own workstation or laptop you can either
 
  - *(recommended)* Use the :ref:`aiidalab-launch <usage:aiidalab-launch>` tool which is a thin docker wrapper, or
- - run the image directly with: ```docker run -p 8888:8888 aiidalab/aiidalab-docker-stack```.
+ - run the image directly with: ``docker run -p 8888:8888 aiidalab/full-stack``.
 
 To use **AiiDAlab launch** you will have to
 
 #. `Install Docker on your workstation or laptop. <https://docs.docker.com/get-docker/>`_
+
+.. note::
+
+   If you are using Linux, you need to have `root` privileges to
+   do [post-installation steps for Docker Engine](https://docs.docker.com/engine/install/linux-postinstall/).
```

Comment:
Minute nitpick: I think we typically just put [one sentence per line](https://github.com/aiidateam/aiida-core/wiki/Writing-documentation#style-guide) with no manual wrapping. 

Also I think adding a `the` here sounds more natural:

```suggestion
   do [post-installation steps for the Docker Engine](https://docs.docker.com/engine/install/linux-postinstall/).
```

Reply:
Yep, thanks!